src/peft/utils/corda_utils.py

```diff
@@ -0,0 +1,314 @@
+# Copyright 2024-present the HuggingFace Inc. team.
```

Comment:
Since `corda_utils.py`  is specific to LoRA, let's move it to `tuners/lora/`. It can also be renamed to just `corda.py`.